junit-common/src/main/java/org/jboss/weld/junit/MockBean.java

```diff
@@ -270,6 +297,25 @@ private Builder() {
             return this;
         }
 
+        /**
+         * The bean is an alternative and should be automatically selected for the synthetic bean archive.
+         *
+         * <p>
+         * Users are encouraged to specify {@link #beanClass(Class)} when using this method. The bean class is used to determine which alternative beans are
+         * selected for a bean archive. By default, all mock beans share the same bean class.
```

Comment:
I would be more specific and say that by default the bean class is set to that of the extension.

Reply:
Ok.